README.md

```diff
@@ -35,48 +44,28 @@ generator. So in order to create a documentation for an entire project you simpl
 $ typedoc --out path/to/documentation/ path/to/typescript/project/
 ```
 
-### Important note
-
-Starting with version 0.2, TypeDoc no longer can predict whether files should be treated as modules
-or whether the project should be compiled into one big namespace. You must specify the `mode` argument
-in order to change the behaviour of TypeDoc.
-
 ### Arguments
 
-For a complete list of the command line arguments run `typedoc --help` or read [here](https://typedoc.org/guides/arguments/).
+For a complete list of the command line arguments run `typedoc --help` or visit [our websit](https://typedoc.org/guides/arguments/).
 
 * `--out <path/to/documentation/>`<br>
-  Specifies the location the documentation should be written to.
+  Specifies the location the documentation should be written to. Defaults to `./docs`
 * `--mode <file|modules>`<br>
-  Specifies the output mode the project is used to be compiled with.
+  Specifies the output mode the project is used to be compiled with. Required since verion 0.2.
```

Comment:
This isn't true. The mode defaults to modules. It might be best practice to specify it as we might want to change the default if the library mode is ever completed, but it isn't required.

https://github.com/TypeStrong/typedoc/blob/cdf3acdd5e56b4fb6683f2ddb8067220e44dde1b/src/lib/converter/nodes/block.ts#L22-L32